packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -313,7 +342,209 @@ const ToggleMenuDrawer = React.forwardRef(
+                                                        'header__menu--back-to-ui': !is_dark_mode,
+                                                        'header__menu--back-to-ui--dark': is_dark_mode,
+                                                    })}
+                                                    type='button'
+                                                    large
+                                                    onClick={() => {
+                                                        setIsPreAppStore(false);
+                                                        toggleDrawer();
+                                                    }}
+                                                >
+                                                    <Text
+                                                        className={classNames({
+                                                            'header__menu--back-to-ui-text': !is_dark_mode,
+                                                            'header__menu--back-to-ui-text--dark': is_dark_mode,
+                                                        })}
+                                                        size='xs'
+                                                    >
+                                                        {localize('Back to old interface')}
+                                                    </Text>
+                                                    <Icon
+                                                        className='header__menu-mobile-right-arrow'
+                                                        icon='IcArrowRight'
+                                                        size={18}
+                                                        color='red'
+                                                    />
+                                                </Button>
+                                            </MobileDrawer.Item>
+                                        )}
+                                        {is_logged_in && (
+                                            <MobileDrawer.Item>
+                                                <MenuLink
+                                                    link_to={routes.trading_hub}
+                                                    icon={is_dark_mode ? 'IcAppstoreHomeDark' : 'IcAppstoreHome'}
+                                                    text={localize('Trading Hub')}
+                                                    onClickLink={toggleDrawer}
+                                                    changeCurrentLanguage={changeCurrentLanguage}
+                                                />
+                                            </MobileDrawer.Item>
+                                        )}
+                                        {is_logged_in &&
+                                            !route.startsWith(routes.trading_hub) &&
```

Comment:
This condition is repeated twice, please declare it on the top of component and reuse it
 const is_trading_hub_category = (route.startsWith(routes.trading_hub) ||
                                            route.startsWith(routes.cashier) ||
                                            route.startsWith(routes.account)